toolchain/lower/testdata/interop/cpp/function_decl.carbon

```diff
@@ -22,6 +22,20 @@ fn MyF() {
   Cpp.foo();
 }
 
+// --- inline_function_decl.h
+
+inline void foo() {}
+
+// --- import_inline_function_decl.carbon
```

Comment:
Why are you adding a non-prelude duplicate test?

Reply:
Sorry, I'm getting this backwards -- this isn't duplicate, but it should definitely have a TODO

Reply:
Added a TODO.

Reply:
```suggestion
// --- todo_import_inline_function_decl.carbon
```

Reply:
Done.
Thanks for clarifying! I wasn't aware that every test with a TODO should have `todo` mark in its name.